winui/Rating/Precision.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Precision with WinUI Rating control | Syncfusion
+description: Learn here all about precision mode defines the accuracy level and it has full, half and exact options of the SfRating control.
```

Comment:
description: Learn about the precision mode, which defines the accuracy level and has full, half, and exact options of the SfRating control here.

Reply:
Changed the content description as mentioned above.